TYPO3.TYPO3CR/Tests/Functional/Domain/NodesTest.php

```diff
@@ -113,14 +113,14 @@ public function nodeCreationThrowsExceptionIfNodeNameContainsUppercaseCharacters
     /**
      * @test
      */
-    public function setPathWorksRecursively()
+    public function setNameWorksRecursively()
     {
         $rootNode = $this->context->getRootNode();
 
         $fooNode = $rootNode->createNode('foo');
         $bazNode = $fooNode->createNode('bar')->createNode('baz');
 
-        $fooNode->setPath('/quux');
+        $fooNode->setName('quux');
```

Comment:
This is the actual change here, unfortunately had to reformat this file, so the other changes are collaterals.